Document extra admin setting when running sandbox with authentication (#345)
digital-asset/docs.daml.com: docs/2.7.0/docs/tools/sandbox.rst

          TLS with server authentication as described in :ref:`sandbox-tls` for
          any request sent to the Ledger API in production.

> Fresh participant nodes come with the default participant admin user called ``participant_admin`` which

Reviewer: ```suggestion
Fresh participant nodes come with a default participant admin user called ``participant_admin`` which
```

can be used to bootstrap other users.
You might prefer to have an admin user with a different user id ready on a participant startup.

> For such situations you can specify an additional participant admin user with the user id of your choice.

Reviewer 2: ```suggestion
For such situations, you can specify an additional participant admin user with the user id of your choice.
```

> Fresh participant nodes come with the default participant admin user called ``participant_admin`` which
> can be used to bootstrap other users.
> You might prefer to have an admin user with a different user id ready on the participant startup.
> For such situations you can specify an additional participant admin user with the user id of your choice.

Reviewer: ```suggestion
Fresh participant nodes come with a default participant admin user called ``participant_admin``, which
can be used to bootstrap other users.
You might prefer to have an admin user with a different user id ready on a participant startup.
For such situations you can specify an additional participant admin user with the user id of your choice.
```

> .. note:: If a user with the specified id already exists then no additional user will be created,

Reviewer 2: ```suggestion
.. note:: If a user with the specified id already exists, then no additional user will be created,
```

>           even if the pre-existing user was not an admin user.

Reviewer 2: ```suggestion
          even if the preexisting user was not an admin user.
```
